src/app/store/EN_15804.json

```diff
@@ -117,7 +117,7 @@
             "type": "LCIA",
             "group": "ENVIRONMENTAL",
             "code": "EP",
-            "unit": "kg (PO4)3- eq.",
+            "unit": "kg P-eq.",
```

Comment:
I switch from `(PO4)3-` to `P` (for phosphate) as it is also used as `(PO4)3--eq.` or `((PO4)3-)-eq.` seems too heavy.